application/src/main/java/org/opentripplanner/graph_builder/issues/DifferentLevelsSharingBarrier.java

```diff
@@ -0,0 +1,21 @@
+package org.opentripplanner.graph_builder.issues;
+
+import org.locationtech.jts.geom.Geometry;
+import org.opentripplanner.framework.geometry.GeometryUtils;
+import org.opentripplanner.graph_builder.issue.api.DataImportIssue;
+import org.opentripplanner.osm.model.OsmNode;
+
+public record DifferentLevelsSharingBarrier(OsmNode node) implements DataImportIssue {
+  private static final String FMT =
+    "Node %d is a barrier node but ways on different layers / levels are connected to it. Please check if the barrier actually blocks traversal, if not, please remove the node from the barrier.";
+
```

Comment:
Please add an OSM link to the issue, so that it is easier to check the node:

```
@Override
  public String getHTMLMessage() {
    return String.format(HTMLFMT, node.url(), node.getId());
  }
```